tutorials/3d/portals/fundamentals.rst

```diff
@@ -0,0 +1,102 @@
+Fundamentals
+============
+The RoomManager
+~~~~~~~~~~~~~~~
+Anytime you want to use the portal system, you need to include a special node in your scene tree, called the :ref:`RoomManager<class_RoomManager>`. The RoomManager is responsible for the runtime maintenance of the system, especially converting the objects in your rooms into a `room graph` which is used at runtime to perform occlusion culling and other tasks.
+
+Room Conversion
+^^^^^^^^^^^^^^^
+This conversion must take place every time you want to activate the system, it does not store the `room graph` in your project (for flexibility and to save memory). You can either trigger it by pressing the ``Convert Rooms`` button in the editor toolbar when the RoomManager is selected, or you can call the ``rooms_convert`` function in the RoomManager. This latter method will be what you use in game. Note that for safety, best practice is to call ``rooms_clear`` before unloading / changing levels.
+
+If you convert the level while the editor is running, the portal culling system will take over from the normal Godot frustum culling. This may affect some editor features. For this reason, you can turn the portal culling on and off, using the ``Active`` setting in the RoomManager.
+
+.. note:: In order use the RoomManager you have to tell it where the rooms are in your scene tree, or rather where the ``RoomList`` node is, that is, the parent of your rooms - see below. If the RoomList is not set, conversion will fail, and you will see a warning dialog box.
+
+.. image:: img/room_manager.png
+
+Rooms
+~~~~~
+What is a room?
+^^^^^^^^^^^^^^^
+:ref:`Room<class_Room>`\ s are a way of spatially partitioning your level into areas that make sense in terms of the level design. Rooms often quite literally *are* rooms (for instance in a building). Ultimately though, as far as the engine is concerned, a room respresents a **non-overlapping** convex volume, in which you would typically place most of your objects that fall within that area.
+
+A room doesn't need to correspond to a literal room. It could for example also be a canyon in an outdoor area, or a smaller part of a concave room. With a little imagination you can use the system in almost any scenario.
```

Comment:
```suggestion
A room doesn't need to correspond to a literal room. It could for example also be a canyon in an outdoor area, or a smaller part of a concave room. With a little imagination, you can use the system in almost any scenario.
```